src/views/product-integration/component-view/components/variable-group-list/index.tsx

```diff
@@ -35,32 +39,52 @@ export function VariableGroupList({
 				return (
 					<li
 						key={variable.key}
-						className={classNames({
+						className={classNames(s.variableGroupListItem, {
 							[s.highlight]: variable.highlight,
 						})}
 					>
-						<code className={s.key}>
-							<strong>{variable.key}</strong>
-						</code>
-						<code className={s.type}>{variable.type}</code>
-						{typeof variable.description !== 'undefined' && (
-							<div>
+						{isNested ? (
+							<div className={s.arrowIcon}>
+								<IconCornerDownRight16 />
+							</div>
+						) : (
+							<></>
+						)}
+						<div className={s.indentedContent}>
+							<div className={s.topRow}>
+								<span className={s.left}>
+									<code className={s.key}>{variable.key}</code>
+									{variable.required != null && (
+										<span
+											className={classNames(s.required, {
+												[s.isRequired]: variable.required,
+											})}
+										>
+											{variable.required ? 'Required' : 'Optional'}
+										</span>
+									)}
+								</span>
+								{variable.type ? (
+									<Badge color="highlight" text={variable.type} />
+								) : (
+									<></>
+								)}
```

Comment:
Same nit: [logical &&](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) instead of ternary + fragment

Reply:
Going to resolve this as the case is not out on the preference here.  Will adjust once we land on something as a team!